allennlp/fairness/bias_mitigator_applicator.py

```diff
@@ -0,0 +1,115 @@
+"""
+A Model wrapper to mitigate biases in
+contextual embeddings during finetuning
+on a downstream task and test time.
+
+Based on: Dev, S., Li, T., Phillips, J.M., & Srikumar, V. (2020).
+[On Measuring and Mitigating Biased Inferences of Word Embeddings]
+(https://api.semanticscholar.org/CorpusID:201670701).
+ArXiv, abs/1908.09369.
+"""
+
+from overrides import overrides
+
+from allennlp.fairness.bias_mitigator_wrappers import BiasMitigatorWrapper
+
+from allennlp.common.lazy import Lazy
+from allennlp.data import Vocabulary
+from allennlp.models import Model
+from allennlp.nn.util import find_embedding_layer
+
+
+@Model.register("bias_mitigator_applicator")
+class BiasMitigatorApplicator(Model):
+    """
+    Wrapper class to apply bias mitigation to any pretrained Model.
+
+    # Parameters
+
+    vocab : `Vocabulary`
+        Vocabulary of base model.
+    base_model : `Model`
+        Base model for which to mitigate biases.
+    bias_mitigator : `Lazy[BiasMitigatorWrapper]`
+        Bias mitigator to apply to base model.
+    """
+
+    def __init__(
+        self,
+        vocab: Vocabulary,
+        base_model: Model,
+        bias_mitigator: Lazy[BiasMitigatorWrapper],
+        **kwargs
+    ):
+        super().__init__(vocab, **kwargs)
+
+        self.base_model = base_model
+        # want to keep bias mitigation hook during test time
+        embedding_layer = find_embedding_layer(self.base_model)
+
+        self.bias_mitigator = bias_mitigator.construct(embedding_layer=embedding_layer)
+        embedding_layer.register_forward_hook(self.bias_mitigator)
+
+        self.vocab = self.base_model.vocab
+        self._regularizer = self.base_model._regularizer
+        self.serialization_dir = self.base_model.serialization_dir
```

Comment:
This is actually unnecessary since the `Model` base class already has `self.serialization_dir` set.
```suggestion
```